### PR TITLE
[Merged by Bors] - Release 0.10.3

### PR DESCRIPTION
**Pull request by davidbeesley**

### CODE REVIEW

**Review by nacardin — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Release Notes
 
-## Platform Version 0.10.3 - Unreleased
+## Platform Version 0.10.3 - 2022-12-16
+* Add `fluvio cluster status` ([#2824](https://github.com/infinyon/fluvio/issues/2824))
+* Display fetch status ([#2872](https://github.com/infinyon/fluvio/issues/2872))
+* Emphasize flush in docs ([#2850](https://github.com/infinyon/fluvio/issues/2850))
```
Comment (sehz) [inline CHANGELOG.md:6]: bunch of other changes are not reflected here

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-0.10.3-dev
```
Comment (sehz) [inline VERSION:1]: why is this was called `dev` not `dev-1`?

**Review by sehz — CHANGES_REQUESTED** [review]

File: VERSION (hunk 2 above)
Comment (davidbeesley) [reply VERSION:1]: That was done https://github.com/infinyon/fluvio/commit/673e60c013f0f0e351ba335c5e7d462711b146b2

Comment (sehz) [reply VERSION:1]: that didn't follow the release doc.  @tjtelan  we need to automate this.

**Review by morenol — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (morenol) [reply CHANGELOG.md:6]: Added entry for smdk fix and for smartengine metrics fix.

**Review by sehz — APPROVED** [review]

**Pull request closed without merging**